ci-operator/step-registry/ipi/deprovision/gcp/aws-route53/ipi-deprovision-gcp-aws-route53-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: ipi-deprovision-gcp-aws-route53
+  from_image:
+    namespace: ocp
+    name: "4.15"
+    tag: upi-installer
+  commands: ipi-deprovision-gcp-aws-route53-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 100Mi
+  credentials:
+  - namespace: ci
+    name: cluster-secrets-aws-qe
```

Comment:
I am afraid this secret might be abused and some security risk. @gpei @yunjiang29 wdyt ?